packages/io/stl-deserializer/test/test.js

```diff
@@ -128,79 +131,95 @@ polyhedron({ points: [
 })
 
-test('deserialize medium complexity binary stl to cag/csg objects', function (t) {
+test('deserialize medium complexity binary stl to geometry', function (t) {
   const inputPath = path.resolve(samplesPath, 'stl/pr2_head_tilt.stl')
   const inputFile = fs.readFileSync(inputPath)
 
-  const observed = deserializer.deserialize(inputFile, undefined, {output: 'csg', addMetaData: false})
-  t.deepEqual(observed.polygons.length, 849)
-  const observedVertices = polygonsFromCsg(observed)
-  const expectedVertices = JSON.parse(fs.readFileSync(path.join(__dirname, 'pr2_head_tilt_vertices.json'), 'utf8'))
-  t.deepEqual(observedVertices, expectedVertices)
+  const observed = deserializer.deserialize(inputFile, undefined, { output: 'csg', addMetaData: false })
+  t.is(observed.length, 1)
+  const polygons = geometry.geom3.toPolygons(observed[0])
+  t.deepEqual(polygons.length, 1052)
 })
 
-test('deserialize complex ascii stl to cag/csg objects', function (t) {
+test('deserialize complex ascii stl to geometry', function (t) {
   const inputPath = path.resolve(samplesPath, 'stl/herringbone-gear-large.stl')
   const inputFile = fs.readFileSync(inputPath, 'utf8')
 
-  const observed = deserializer.deserialize(inputFile, undefined, {output: 'csg', addMetaData: false})
-  t.deepEqual(observed.polygons.length, 14611)
-  // too many vertices for a reasonable in detail test : even a generated JSON from just the CSG vertices is almost 2 MB !
-
-  // const observedVertices = polygonsFromCsg(observed)
-  // const expectedVertices = JSON.parse(fs.readFileSync('./testExpecteds/herringbone-gear-large.json', 'utf8'))
-  // t.deepEqual(observedVertices, expectedVertices)
-  // fs.writeFileSync('herringbone-gear-large.json', JSON.stringify(observedVertices))
+  const observed = deserializer.deserialize(inputFile, undefined, { output: 'csg', addMetaData: false })
+  t.is(observed.length, 1)
+  const polygons = geometry.geom3.toPolygons(observed[0])
+  t.deepEqual(polygons.length, 17742)
 })
 
-test('deserialize complex binary stl to cag/csg objects (2)', function (t) {
+test('deserialize complex binary stl to geometry', function (t) {
   const inputPath = path.resolve(samplesPath, 'stl/UM2CableChain_BedEnd.STL')
   const inputFile = fs.readFileSync(inputPath)
 
-  const observed = deserializer.deserialize(inputFile, undefined, {output: 'csg', addMetaData: false})
-  t.deepEqual(observed.polygons.length, 10241)
-  // too many vertices for a reasonable in detail test : even a generated JSON from just the CSG vertices is almost 2 MB !
-
-  // const observedVertices = polygonsFromCsg(observed)
-  // const expectedVertices = JSON.parse(fs.readFileSync('./testExpecteds/herringbone-gear-large.json', 'utf8'))
+  const observed = deserializer.deserialize(inputFile, undefined, { output: 'csg', addMetaData: false })
+  t.is(observed.length, 1)
+  const polygons = geometry.geom3.toPolygons(observed[0])
+  t.deepEqual(polygons.length, 12744)
```

Comment:
how come the polygons length has changed so much . should it not be around the same as before ?